packages-ts/gauntlet-terra/src/commands/internal/terra.ts

```diff
@@ -110,10 +113,11 @@ export default abstract class TerraCommand extends WriteCommand<TransactionRespo
     return this.wrapResponse(res)
   }
 
-  async deploy(codeId, instantiateMsg, migrationContract = undefined) {
+  async deploy(codeId, instantiateMsg, admin = this.wallet.key.accAddress, definitive = false) {
+    logger.info(`Deploying contract ${definitive ? 'marked as definitive' : 'with admin ' + admin}`)
     const instantiate = new MsgInstantiateContract(
       this.wallet.key.accAddress,
-      migrationContract || this.wallet.key.accAddress,
+      definitive ? undefined : admin,
```

Comment:
This `definitive` arg is not used anywhere currently, right? Please map the `deploy` cmd flag to this arg so it's connected.

Also consider renaming as `final` or `nonUpgradeable` which could be more readable and understandable flags in this context.

Reply:
Yeah right, is not used. Let me expose it. I'll leave it as `final`